packages/editor/src/store/selectors.js

```diff
@@ -1140,7 +1156,7 @@ const isAncestorOf = createSelector(
 		return possibleAncestorId === idToCheck;
 	},
 	( state ) => [
-		state.editor.present.blocks,
+		state.editor.present.blocks.order,
```

Comment:
Are these revisions relevant to the proposed change? Or are these things which are technically separate issues?

Reply:
Actually, these changes are one of the important changes in this PR as we don't invalidate these selectors unless the inserted blocks (order) change instead of any attribute change.

Reply:
I can see it would have an important impact on the overall benefit of the memoization; but is it specifically related to pulling out attributes to a separate state path?

Reply:
Actually, you're right, this one in particular is not directly related.

Reply:
I'm happy enough to create a separate PR for this, but it doesn't seem like it would be outlandish to keep it in this PR either.